common/arch/win32/rbaudio.cpp

```diff
@@ -392,92 +417,69 @@ int RBAPeekPlayStatus()
     mciStatusParms.dwItem = MCI_STATUS_MODE;
     if ((mciError = mciSendCommand(wCDDeviceID, MCI_STATUS, MCI_STATUS_ITEM, reinterpret_cast<DWORD_PTR>(&mciStatusParms))))
     {
-		Warning("RBAudio win32: cannot determine MCI media status (%lx)", mciError);
+        Warning("RBAudio win32/MCI: cannot determine MCI media status (%lx)", mciError);
         return 0;
     }
 
-	if (mciStatusParms.dwReturn == MCI_MODE_PLAY)
+    if (mciStatusParms.dwReturn == MCI_MODE_PLAY)
         return 1;
-	else if (mciStatusParms.dwReturn == MCI_MODE_PAUSE)
-		return -1; // hack so it doesn't keep restarting paused music
-	else
-		return 0;
+    else if (mciStatusParms.dwReturn == MCI_MODE_PAUSE)
+        return -1; // hack so it doesn't keep restarting paused music
```

Comment:
Please document why `-1` is a hack that has this effect.  If possible, avoid using a hack at all, and instead structure the logic so that paused music stays paused without this special case.

Reply:
This is also used in the SDL CD code.